cellprofiler/gui/pipelinecontroller.py

```diff
@@ -1166,7 +1166,7 @@ def __on_plateviewer(self, event):
         if len(image_numbers) == 0:
             self.display_plate_viewer_help(
```

Comment:
You may also want to change the control defined in `display_plate_viewer_help` to support links.